cucumber/api/features/login.feature

```diff
@@ -5,12 +5,87 @@ Feature: Exchange a role's password for its API key
 
   Background:
     Given I create a new user "alice"
+    Given I have host "app"
 
   Scenario: Password can be used to obtain API key
     Given I set the password for "alice" to "My-Password1"
     When I can GET "/authn/cucumber/login" with username "alice" and password "My-Password1"
     Then the HTTP response content type is "text/plain"
     And the result is the API key for user "alice"
+    And there is an audit record matching:
+    """
+      <84>1 * * conjur * authn
+      [auth@43868 authenticator="authn" service="default:webservice:conjur/authn]
+      [subject@43868 role="cucumber:user:alice" user="alice"]
+      [action@43868 operation="login" result="success"]
+      cucumber:user:alice successfully logged in with authenticator authn service default:webservice:conjur/authn
+    """
+  Scenario: Password can be used by host to obtain API key
+    Given I set the password for "host/app" to "My-Password1"
+    When I can GET "/authn/cucumber/login" with username "host/app" and password "My-Password1"
```

Comment:
what is the test here? do we test that we can get? if so, please change this to `Then i can get`. If not, and the test is only on the output then this should be `And I can get` (because it's given)

Reply:
The 2nd option. I'll change